utopia-remix/app/routes-test/internal.projects.$id.access.request.$token.destroy.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { prisma } from '../db.server'
+import { handleDestroyAccessRequest } from '../routes/internal.projects.$id.access.request.$token.destroy'
```

Comment:
If you want you can also test `action` itself, to test for permission (that it doesn't allow if you don't have the proper permission).